### DISCUSSION TITLE
Feeding Large Dataset to Qdrant Client

### DISCUSSION THREAD

**Discussion by harshakhmk** [qdrant/qdrant#3330] [General]
I am looking to experiment with feeding large datasets such as [MNIST](http://ann-benchmarks.com/mnist-784-euclidean.hdf5) or [NYTimes](http://ann-benchmarks.com/nytimes-256-angular.hdf5) to Qdrant Client, the file format is mostly in hdf5 file format 
Any leads on how to proceed would be highly appreciated

**Comment by generall** [comment]
Some relevant links:

https://github.com/qdrant/fastembed

https://github.com/qdrant/qdrant_demo/blob/master/qdrant_demo/init_collection_crunchbase.py

**Reply by harshakhmk** [reply]
How to read and upload the data to Qdrant server from large datasets like MNIST, NyTimes whose data is in .hdf5 format

**Reply by harshakhmk** [reply]
Hi any suggestions on how to proceed ahead 
Thanks in advance
